semester-two/mcsl209/03-linked-list/find-common-elements.c

```diff
@@ -94,37 +98,39 @@ void displayList( struct Node *firstNode ) {
     
 }
 
-struct Node* createListWithCommonNodes( struct Node* firstNodeListA, struct Node* firstNodeListB, struct Node* firstNodeListC ) {
+struct Node* createListWithCommonNodes( struct Node* firstNodeListA, struct Node* firstNodeListB ) {
```

Comment:
@sumayyahc1 Can you please add the doc comment with param and return type?

Reply:
Done.